Mighty-Menorahs/reactive-nutshell
# file structure

# Description

Added File Structures for Modules and Setup Individual Component Folders and Api Handler Folder.



Please delete options that are not relevant.



# Testing Instructions

-git fetch --all
-git checkout jbFirstLook
-run code.
-check for new folders in vscode



# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
